src/Teams/Controllers/MultipleAnswerQuestionController.cs

```diff
@@ -37,5 +37,59 @@ public IActionResult MultipleAnswerQuestionForm(string id, int[] answers)
             };
             return View(question);
         }
+
+        public IActionResult EditMultipleAnswerQuestion(Guid id)
+        {
+            var question = new MultipleAnswerQuestionViewModel()
+            {
+                SourceQuestion = questionRepository.PickById(id),
+                // ChosenOptions = answers
+            };
+            return View(question);
+        }
+
+        public IActionResult AddMultipleAnswerQuestion()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult AddMultipleAnswerQuestion([FromBody] MAQDTOModel fromAjax)
+        {
+            List<MultipleAnswerQuestionOption> allAnswers = new List<MultipleAnswerQuestionOption>();
+            for (int i = 0; i < fromAjax.jTextList.Length; i++)
+            {
+                allAnswers.Add(new MultipleAnswerQuestionOption(fromAjax.jTextList[i], fromAjax.jCheckboxList[i]));
+            }
+            questionRepository.MethodForAdd(new MultipleAnswerQuestion(fromAjax.questionText, allAnswers));
+
+            return View(); 
+            
+        }
+        [HttpPost]
+        public IActionResult EditMultipleAnswerQuestion( int[] editAnswers, string[] answersText, string questionText, Guid Id)
+        {
+            List<MultipleAnswerQuestionOption> allAnswers = new List<MultipleAnswerQuestionOption>();
+            int g = 0;
```

Comment:
What is the meaning of the "g" variable. Could you use more meaningful names for variables?